### PR TITLE
Milestone draft

### PR DESCRIPTION
**Pull request by sshiells-scottlogic**
Add first draft of release guidelines.

Covers:

- Release ownership
- Scheduling
- Project board management

### CODE REVIEW

File: docs/governance/community-guidelines/releases.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This document is a [community guideline].
```
Comment (dogle-scottlogic) [inline docs/governance/community-guidelines/releases.md:3]: Links are giving me a 404 but that might just be the GitHub preview thing

File: docs/governance/community-guidelines/releases.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the CCC project.
+
+## Ownership
+
+The [Communications WG] takes ownership of releases. They set the roadmap and communicate project progress to the wider community. All other [WG]s should contribute to determining the content of each release.
```
Comment (dogle-scottlogic) [inline docs/governance/community-guidelines/releases.md:11]:
```suggestion
The [Communications WG] has overall ownership of releases. They are responsible for setting the roadmap and communicating project progress to the wider community. All other [WG]s should contribute to determining the content of a release.
```

File: docs/governance/community-guidelines/releases.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the CCC project.
```
Comment (dogle-scottlogic) [inline docs/governance/community-guidelines/releases.md:7]: Should probably use "Common Cloud Controls (CCC)" the first time and the acronym for all the others

File: docs/governance/community-guidelines/releases.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Ownership
+
+The [Communications WG] takes ownership of releases. They set the roadmap and communicate project progress to the wider community. All other [WG]s should contribute to determining the content of each release.
+
+## Scheduling
+
+The community aims for several releases per year. Releases can occur as soon as the controls for a service are complete and approved. The CCC project should prioritize stable releases and where possible avoid excessive release frequency (e.g., multiple times per day).
```
Comment (dogle-scottlogic) [inline docs/governance/community-guidelines/releases.md:15]: There's quite a range between "several a year" vs "multiple per day". More an observation than anything else

Comment (dogle-scottlogic) [reply docs/governance/community-guidelines/releases.md:15]: What constitutes stable in this context?

File: docs/governance/community-guidelines/releases.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+### Events
+
+The community can align releases with events such as the Open Source in Finance Forum (OSFF). These events provide excellent opportunities to showcase the project and generating interest.
```
Comment (dogle-scottlogic) [inline docs/governance/community-guidelines/releases.md:19]: worth adding a link for OSFF?

**Review by dogle-scottlogic — COMMENTED** [review]
Is it worth documenting how a release is cut and what versioning we use etc?

File: docs/governance/community-guidelines/releases.md (hunk 1 above)
Comment (sshiells-scottlogic) [reply docs/governance/community-guidelines/releases.md:3]: Works for me locally (i.e. it opens the correct page in VS code when clicking on it) so I presume it is a preview thing.

File: docs/governance/community-guidelines/releases.md (hunk 4 above)
Comment (sshiells-scottlogic) [reply docs/governance/community-guidelines/releases.md:15]: not multiple releases per day :)

File: docs/governance/community-guidelines/releases.md (hunk 1 above)
Comment (dogle-scottlogic) [reply docs/governance/community-guidelines/releases.md:3]: Yeah - link-checker is still showing the same amount of fails as before so should be good 👍

**Comment by sshiells-scottlogic** [comment]
> Is it worth documenting how a release is cut and what versioning we use etc?

We will when we know :D

**Comment by dogle-scottlogic** [comment]
looks good to me

File: docs/governance/community-guidelines/releases.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+### Labels
+
+Each [WG] should apply their label to relevant issues, clearly indicating which issues fall under their responsibility.
+
+Additionally, labels can flag issues specifically targeted for events (e.g., OSFF events).
+
+[WG]: <../community-groups.md#working-groups>
+[Communications WG]: <../working-groups/communications/charter.md>
+[FINOS Point of Contact]: <../finos-poc.md>
```
Comment (damienjburks) [inline docs/governance/community-guidelines/releases.md:42]: Can we remove this if it's not going to be used?

**Review by damienjburks — CHANGES_REQUESTED** [review]

**Review by eddie-knight — APPROVED** [review]
LGTM, thanks @sshiells-scottlogic

**Comment by damienjburks** [comment]
@sshiells-scottlogic are you able to configure and execute prettier using the docs here: 

https://github.com/finos/common-cloud-controls/blob/main/docs/training/lint_format_user_guide.md

**Comment by eddie-knight** [comment]
@damienjburks I'm becoming a bit concerned that contributors need to set up a local node development environment for modifying markdown files. Do we have an alternative that we can explore?

**Comment by damienjburks** [comment]
@eddie-knight there is an alternative method. I'll raise a PR for it in a few hrs for you to review.

**Review by damienjburks — APPROVED** [review]

**Comment by damienjburks** [comment]
@eddie-knight i'll approve and merge in anyway. Don't want to halt up the progress.

**Comment by sshiells-scottlogic** [comment]
Thanks for merging in. Apologies for the lack of reply. I have had issues with emails on my phone.

I have had some difficulties in getting things set up properly. I only have my work laptop and I was selected to trial some new increased secuity setup (and I can tell you it is pretty secure :D)

**Pull request merged**